README.md

```diff
@@ -27,11 +27,41 @@ You also will need to install `tezos-wallet` app for initial setup to stake tez
   - `IMPORTANT` Make sure to `enable screensaver` and `disable global PIN lock` in ledger settings.
   - Open the baking app and start baking with octez client.
 
-Disabling global PIN lock makes it possible for baking app to continue respond to signing requests even when screensaver is running on the screen of the ledger. Baking app uses Ledger screensaver to avoid screen burn.
+Disabling global PIN lock makes it possible for baking app to continue respond to signing requests even when screensaver is running on the screen of the ledger. Baking app uses Ledger screensaver (custom screensaver for nanos) to avoid screen burn.
 
 The baking app once opened can not be exited until you enter PIN again so disabling global PIN lock while baking does not pose any risk.
 
 `Caution` Make sure that a dedicated Ledger device is used for baking and not to leave any other app open in screensaver mode while global PIN lock is disabled.
+
+## HWM settings
+
+To avoid double baking, double attestation and double pre-attestation, the application maintains a high water mark (HWM) corresponding to the last level/round encountered during signature requests. The HWMs are displayed on the home screen and are updated after each signature.
+
+For performance reasons, the HWM screen is not updated dynamically. On Nano devices, press both buttons to update.
+
+To make sure HWM values are preserved after a reboot/power_off, the HWM values are saved in non-volatile memory(NVRAM) on ledger device.
+
+Previously `baking_app` would store every new value to NVRAM, thus causing NVRAM burn. NVRAM has limited number of write cycles after which it stops working.
+Therefore, in the new `baking_app` we have added a setting to disable HWM. By default it is enabled.
+
+When HWM setting is disabled, the HWM will not be written to NVRAM on every block level/round, rather it will only be updated in RAM on every level/round. Only when you exit the app properly by clicking `Quit`, will the latest HWM value be written to NVRAM.
```

Comment:
I do not understand `on every block level/round` or `on every level/round`. HWM is written on every, block, pre-attestation or attestation.

Reply:
How about 
" When HWM setting is disabled, the HWM will be written to RAM instead of NVRAM after every signing operation."